tests/test_triton_server.py

```diff
@@ -39,7 +40,12 @@
 }
 
 
+def get_gpus_uuids(gpus_config):
+    return GPUDeviceFactory.verify_requested_gpus(gpus_config)
```

Comment:
I think you need to mock the `GPUDeviceFactory`.

Reply:
Good suggestion. This speeds up the test. It is updated.